src/components/TipsLayout/TipsLayout.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+//style
+import "./TipsLayout.sass";
+//components
+import { Container } from "@material-ui/core";
+
+import { Header } from "../Header/Header";
```

Comment:
Cambiar a ruta absoluta

```suggestion
import { Header } from "components/Header/Header";
```